packages/nvidia_nat_rag_lib/src/nat/plugins/rag_lib/client.py

```
@@ -0,0 +1,216 @@
+        vlm_to_llm_fallback=pipeline.vlm_to_llm_fallback,
+        default_confidence_threshold=pipeline.default_confidence_threshold,
+    )
+
+    # resolve LLM config
+    nim_llm_config = builder.get_llm_config(config.llm)
+    if not isinstance(nim_llm_config, NIMModelConfig):
+        raise ValueError(f"Unsupported LLM config type: {type(config.llm)}. Expected NIMModelConfig.")
+
+    base_dict = nim_llm_config.model_dump(include={"base_url", "model_name", "api_key"}, exclude_none=True)
+    if "base_url" not in base_dict:
+        raise ValueError("base_url is required for LLM config specified in in NVIDIA RAG Config.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)